extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -872,7 +872,6 @@ export class VisibilityManagerForDoc extends VisibilityManager {
         Number(intersection.width),
         Number(intersection.height)
       );
-      // TODO(#29618): Remove after ampim investigation
```

Comment:
Is there any more cleanup needed?

Reply:
Yes. We need to move all the related error logs. 
I removed this TODO because `boundingClientRect` needs to stay because we need element's width/height value.